### PR TITLE
Format haxs

### PR DESCRIPTION
**Pull request by kevmoo**
[Review on Reviewable](https://reviewable.io/reviews/kevmoo/sdk/1)

### CODE REVIEW

File: pkg/analyzer/lib/src/generated/error_verifier.dart [hunk 1]
```diff
@@ -1513,8 +1516,10 @@ class ErrorVerifier extends RecursiveAstVisitor<Object> {
               }
               if (!result.equalValues(_typeProvider, overriddenResult)) {
                 _errorReporter.reportErrorForNode(
-                    StaticWarningCode.INVALID_OVERRIDE_DIFFERENT_DEFAULT_VALUES_NAMED,
-                    formalParameters[i], [
+                    StaticWarningCode
```
Comment (kevmoo) [inline pkg/analyzer/lib/src/generated/error_verifier.dart:1519]: This feels a tiny bit weird...

File: pkg/analyzer/lib/src/generated/error_verifier.dart [hunk 2]
```diff
@@ -3735,11 +3744,10 @@ class ErrorVerifier extends RecursiveAstVisitor<Object> {
...
             _errorReporter.reportErrorForNode(
-                StaticWarningCode.INSTANCE_METHOD_NAME_COLLIDES_WITH_SUPERCLASS_STATIC,
-                errorNameTarget, [
-              executableElementName,
-              methodElement.enclosingElement.displayName
-            ]);
+                StaticWarningCode
+                    .INSTANCE_METHOD_NAME_COLLIDES_WITH_SUPERCLASS_STATIC,
+                errorNameTarget,
+                [executableElementName, methodElement.enclosingElement.displayName]);
```
Comment (kevmoo) [inline pkg/analyzer/lib/src/generated/error_verifier.dart]: BETTER!

File: pkg/analyzer/lib/src/generated/static_type_analyzer.dart [hunk 3]
```diff
@@ -1657,10 +1657,9 @@ class StaticTypeAnalyzer extends SimpleAstVisitor<Object> {
       // the function is being called on] to get a more accurate returnType type
       InterfaceType interfaceTypeContext = context;
       //      Type[] argumentTypes = interfaceTypeContext.getTypeArguments();
-      List<TypeParameterElement> typeParameterElements =
-          interfaceTypeContext.element != null
-              ? interfaceTypeContext.element.typeParameters
-              : null;
+      List<TypeParameterElement> typeParameterElements = interfaceTypeContext
+              .element !=
```
Comment (kevmoo) [inline pkg/analyzer/lib/src/generated/static_type_analyzer.dart]: This feels worse!

File: pkg/analyzer/lib/src/generated/testing/element_factory.dart [hunk 4]
```diff
@@ -194,8 +194,8 @@ class ElementFactory {
   }
 
   static ExportElementImpl exportFor(LibraryElement exportedLibrary,
-      [List<NamespaceCombinator> combinators =
-          NamespaceCombinator.EMPTY_LIST]) {
+      [List<NamespaceCombinator> combinators = NamespaceCombinator
+          .EMPTY_LIST]) {
```
Comment (kevmoo) [inline pkg/analyzer/lib/src/generated/testing/element_factory.dart]: feels worse

File: pkg/analyzer/test/generated/ast_test.dart [hunk 5]
```diff
@@ -1338,7 +1339,8 @@ class SimpleStringLiteralTest extends ParserTestCase {
         5);
     expect(
         new SimpleStringLiteral(
-            TokenFactory.tokenFromString("r'''  \nX'''"), "X").contentsEnd,
+                TokenFactory.tokenFromString("r'''  \nX'''"), "X")
+            .contentsEnd,
```
Comment (kevmoo) [inline pkg/analyzer/test/generated/ast_test.dart:1343]: feels really weird

Comment (munificent) [inline pkg/analyzer/test/generated/ast_test.dart:1343]: This is weird, but as intended.

File: pkg/analyzer/lib/src/generated/error_verifier.dart (hunk 1 above)
Comment (munificent) [inline pkg/analyzer/lib/src/generated/error_verifier.dart:1519]: Yeah, there are some still remaining issues around collections that this is tickling. I wouldn't worry about it for now.

File: pkg/analyzer/lib/src/generated/static_type_analyzer.dart (hunk 3 above)
Comment (munificent) [inline pkg/analyzer/lib/src/generated/static_type_analyzer.dart]: Yeah, can you file a bug for this?

File: pkg/analyzer/lib/src/generated/testing/element_factory.dart (hunk 4 above)
Comment (munificent) [inline pkg/analyzer/lib/src/generated/testing/element_factory.dart]: And this too.

File: pkg/analyzer/lib/src/generated/static_type_analyzer.dart (hunk 3 above)
Comment (kevmoo) [inline pkg/analyzer/lib/src/generated/static_type_analyzer.dart]: done! https://github.com/dart-lang/dart_style/issues/436

File: pkg/analyzer/lib/src/generated/testing/element_factory.dart (hunk 4 above)
Comment (kevmoo) [inline pkg/analyzer/lib/src/generated/testing/element_factory.dart]: Done! https://github.com/dart-lang/dart_style/issues/437

**Pull request closed without merging**